src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -339,6 +339,25 @@ const generateTaskRole = (taskRole, labels, config) => {
                   name: 'GANG_ALLOCATION',
                   value: gangAllocation,
                 },
+                // Pass user token to runtime to give runtime permission to call rest server
+                // Actually we should provide service token for kube-runtime and do not let
+                // runtime personate as a real user.
+                {
+                  name: 'PAI_USER_TOKEN',
+                  value: userToken,
+                },
+                {
+                  name: 'PAI_USER_NAME',
+                  value: labels.userName,
+                },
+                {
+                  name: 'PAI_JOB_NAME',
+                  value: labels.jobName,
+                },
```

Comment:
remove

Reply:
Will check this, maybe my testing bed is too old and not contain these envs

Reply:
I believe these envs only can be used in app container, not init container. For init container, we still need these envs